.buildkite/scripts/check-storybook-accessibility.ts

```diff
@@ -0,0 +1,80 @@
+import AxePuppeteer from "axe-puppeteer"
+import puppeteer from "puppeteer"
+
+const getExamples = async page => {
+  const handle = await page.evaluateHandle(() => ({ window, document }))
+  const properties = await handle.getProperties()
+  const windowHandle = properties.get("window")
+  const clientApiHandle = await (await windowHandle.getProperties()).get(
+    "__STORYBOOK_CLIENT_API__"
+  )
+  const getStorybook = await clientApiHandle.evaluate(clientApi =>
+    clientApi.getStorybook()
+  )
+
+  const result = []
+  getStorybook.forEach(story => {
+    const { kind } = story
+    story.stories.forEach(example => {
+      const { name } = example
+      result.push({
+        kind,
+        name,
+      })
+    })
+  })
+  return result
+}
+
+const asyncForEach = async (array, callback) => {
+  for (let index = 0; index < array.length; index++) {
+    await callback(array[index], index, array)
+  }
+}
+
+const getViolations = async (storybookExampleUrls, page) => {
+  const axePuppeteerInstance = new AxePuppeteer(page)
+  const analyses = []
+  await asyncForEach(storybookExampleUrls, async url => {
+    await page.goto(url)
+    const analysis = await axePuppeteerInstance.analyze()
+    if (analysis.violations.length > 0) {
+      analyses.push({ url, violations: analysis.violations })
+    }
+  })
+  return analyses
+}
+
+const main = async () => {
+  const browser = await puppeteer.launch()
+  const page = await browser.newPage()
+  await page.setBypassCSP(true)
+
+  const baseIframeUrl = "http://localhost:57357/iframe.html"
```

Comment:
Yeah I'm not sure either... 🤔

Reply:
I think there are two options here:
1. You don't need an http server for the static build of Storybook. I don't know much about Puppeteer, but if you can run it over a completely static html file then this should work: `const baseIframeUrl = "./storybook/public/index.html"`. 
Failing that, a super lightweight server, like http-server, could be setup for the pipeline. https://www.npmjs.com/package/http-server
2. Wait until the artifacts have been uploaded to s3 (publish step) and run this script over the production assets/branch preview. It would be great to have the build fail before assets are published so this is really, really not ideal .`${KAIZEN_DOMAIN_NAME}${KAIZEN_BASE_PATH}/storybook-static`

Reply:
It looks like a fully qualified file URL is fine, so `file://$(pwd)/storybook/public`